### ISSUE TITLE
Sending command message to the device as an event message 

### ISSUE THREAD

**Issue by fayvaz** [eclipse/hono#604]
Hello -

As I see from the documentation, it looks like the command control API has not been supported yet. For that reason, I would like to do that by means of event API. My proposed message flow is like that:
 
[Northbound app] --> [Hono] --> [MQTT Adapter] --> [MQTT Client @ Device]

Is it possible to do that with the existing capabilities of Hono? Or do you have another idea?

Many thanks,
-fatih

**Comment by sophokles73** [comment]
No, this is not possible. However, the upcoming 0.6 release will include a first implementation of the Command & Control API. So, if you can wait 2 more weeks, you should be able to check out if that works for you.
